package.json

```
@@ -188,23 +187,98 @@
                     "markdownDescription": "Enables code linting for HTML+HUBL and CSS+HUBL languages.\n\n*Requires a valid `hubspot.config.yml` file*"
                 }
             }
-        }
+        },
+        "commands": [
+            {
+                "command": "hubspot.config.selectDefaultAccount",
+                "title": "Hubspot: Select Default Account"
+            },
+            {
+                "command": "hubspot.config.setDefaultAccount",
+                "title": "Set Default Account"
+            },
+            {
+                "command": "hubspot.config.deleteAccount",
+                "title": "Delete Account"
+            },
+            {
+                "command": "hubspot.config.renameAccount",
+                "title": "Rename Account"
+            }
+        ],
+        "menus": {
+            "view/item/context": [
+                {
+                    "command": "hubspot.config.setDefaultAccount",
+                    "when": "viewItem == portal"
+                },
+                {
+                    "command": "hubspot.config.renameAccount",
+                    "when": "viewItem == portal"
+                },
+                {
+                    "command": "hubspot.config.deleteAccount",
+                    "when": "viewItem == portal"
+                }
+            ]
+        },
+        "viewsContainers": {
+            "activitybar": [
+                {
+                    "id": "hubspot-explorer",
+                    "title": "HubSpot",
+                    "icon": "images/hubspot-logo.svg"
+                }
+            ]
+        },
+        "views": {
+            "hubspot-explorer": [
+                {
+                    "id": "hubspot.treedata.portals",
+                    "name": "Portals",
```

```suggestion
                    "name": "Accounts",
```

I think we'll want to call this pane the `Accounts` pane. This is the language used throughout the docs and the HS app to refer to portals generally. So we should stay aligned with that.